cli/tools/coverage/style.css

```diff
@@ -23,8 +23,6 @@ body {
 *,
 *:after,
 *:before {
-  -webkit-box-sizing: border-box;
-  -moz-box-sizing: border-box;
```

Comment:
`box-sizing` is sufficient and well-supported: https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing#browser_compatibility